Week2/homework/week2maartjes-work.js

```diff
@@ -0,0 +1,133 @@
+
```

Comment:
Strange spacing in this file too. Too many blank lines. Needs Prettier formatting.